reports/Makefile

```diff
@@ -22,8 +22,9 @@ report/mentor-answered-watched/since-%/report.json: report/mentor-answered-watch
 
 .SECONDARY:
 report/mentor-answered-watched/since-%/report.csv: report/mentor-answered-watched/since-%/report.json
```

Comment:
Do we add a date for the wildcard here?

Reply:
for now, you would just call

```
report/mentor-answered-watched/today
```

...you need XAPI credentials to run the real report